### PR TITLE
Add DVS signing sample app

### PR DESCRIPTION
**Pull request by MiroslavGatsanoga**

### CODE REVIEW

File: MiraclDvsSigningApp.Core2.0/wwwroot/js/site.js [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿// Write your JavaScript code.
```
Comment (simeon-aladjem) [inline MiraclDvsSigningApp.Core2.0/wwwroot/js/site.js]: Is this file needed at all?

Comment (MiroslavGatsanoga) [reply MiraclDvsSigningApp.Core2.0/wwwroot/js/site.js]: All of these files are generated by Visual Studio when you create a MVC project and it's possible that we don't need them. We may consider removing all the boilerplate code  but with all the apps in all the .Net SDKs it will take some time.
